src/app/Instance/BridgeOverview/Components/OBDashboardTableView.tsx

```diff
@@ -61,13 +62,17 @@ export const OBDashboardTableView = (
                 <Td>
                   <Stack>
                     <StackItem>
-                      <Flex>
-                        <FlexItem style={{ flexBasis: "100px" }}>
+                      <Flex flexWrap={{ default: "nowrap" }}>
+                        <FlexItem>
```

Comment:
Preventing the names from wrapping on multiple lines and relying to Flex props instead of the style one.